src/pages/overlay/components/overlay/settings/Settings.tsx

```diff
@@ -17,16 +17,16 @@ export default function Settings(props: OverlayOptionProps) {
     <Card className={className} title="Extension Settings">
       <ul className={styles.settings}>
         {typeSafeObjectEntries(settings).map(([key, setting]) => {
-          if (setting.devOnly && process.env.NODE_ENV !== "development")
+          if (process.env.NODE_ENV !== "development" && !setting.configurable)
             return null;
```

Comment:
Instead of having this here, I think it'd be cleaner to just update the specific DEV-only setting to have `configurable: process.env.NODE_ENV === "development"`?